app/controllers/openid_connect/logout_controller.rb

```diff
@@ -41,5 +71,11 @@ def logout_params
       end
       params.permit(*permitted)
     end
+
+    def render_404_if_disabled
```

Comment:
The [`RenderConditionConcern` concern](https://github.com/18F/identity-idp/blob/main/app/controllers/concerns/render_condition_concern.rb) was designed for this if we'd want to use it.

Example usage:

https://github.com/18F/identity-idp/blob/bfb4a96ff94170baaa76f03e5b7aa84b10c67fcd/app/controllers/idv/in_person_controller.rb#L5

Reply:
Oh nice thank you, had forgotten about that. Added in [3f87767](https://github.com/18F/identity-idp/pull/7017/commits/3f877674d00e43fb258d6743a268946c0ba9adea)